flumine/execution/simulatedexecution.py

```diff
@@ -49,7 +49,14 @@ def execute_place(
                     order, simulated_response, order_package.package_type
                 )
                 if simulated_response.status == "SUCCESS":
-                    order.executable()
+                    if simulated_response.order_status == "EXECUTABLE":
```

Comment:
Same for this, revert it all if we want to match live execution.

Reply:
Hi Liam, I'm afraid I'm still not fully understanding this.

`betfairexecution.py` appears to automatically set the order status to EXECUTABLE here:

https://github.com/liampauling/flumine/blob/65a33d785350d9cfdfa8a9184372e41c23235d10/flumine/execution/betfairexecution.py#L30-L31

However, after placing an unfilled FOK order, the order status in the instruction report will be EXPIRED. Here is an example from an order I just placed:

`{'status': 'SUCCESS', 'marketId': '1.174497522', 'instructionReports': [{'status': 'SUCCESS', 'instruction': {'selectionId': 26628622, 'limitOrder': {'size': 2.0, 'price': 1.01, 'timeInForce': 'FILL_OR_KILL'}, 'orderType': 'LIMIT', 'side': 'LAY'}, 'betId': '214532880901', 'placedDate': '2020-10-21T19:54:31.000Z', 'averagePriceMatched': 0.0, 'sizeMatched': 0.0, 'orderStatus': 'EXPIRED'}]}
`

Should `betfairexecution.py` be updated to read the orderStatus from `instruction_report`, rather than automatically setting it to EXECUTABLE if it is placed succesfully?

Reply:
Because execution is running in a different thread I have historically tried to keep it simple in terms of logic, so it makes the request and then updates based on the `instruction_report.status` only leaving the order status updates to the main handler within `process.py`